### PR TITLE
Home

### PR DESCRIPTION
**Pull request by bossmonkey98**
Home page Added
Apex UI — Mozilla Firefox 17-04-2022 11_50_15

### CODE REVIEW

**Review by t007rushi — APPROVED** [review]
Looks great go ahead

File: index.html [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title> Apex UI</title>
 </head>
 <body>
-    
+    <header class="navbar">
+        <div id="logo">
+            <a href="index.html">
+                <span><img class="logo-icon" src="./assets/logo/apex-legends-symbol-white.png" alt="A"></span><span class="name">pex UI</span></a>
```
Comment (arjun-computer-geek) [inline index.html]: Give meaningful alt text

File: index.html [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+        </div>
+        <div class="Home">
+            <a href="https://github.com/bossmonkey98"><i class="bi bi-github"></i></a>
+            <a href="https://www.linkedin.com/in/arun-kumar-0b95911aa/"><i class="bi bi-linkedin"></i></a>
```
Comment (arjun-computer-geek) [inline index.html:19]: It could be better if you will give target="_blank" for external links.

File: index.html [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+        </div>
+    </header>
+    <div class="content-container">
+        <p>
+           Now design your website easily and faster with <strong style="font-family: Georgia, 'Times New Roman', Times, serif; color:cyan;">Apex UI</strong>
```
Comment (arjun-computer-geek) [inline index.html:24]: Avoid using inline style

File: index.html [hunk 4]
```diff
@@ -1,11 +1,32 @@
...
+        </p>
+        <div>
+            <a href="docs.html"><button class="btn-1">Documentation</button></a>
+            <button class="btn-2">Installation</button>
```
Comment (arjun-computer-geek) [inline index.html:28]: Give a meaningful class name it don't look btn-2 or 3. Provide like btn-explore or primary or secondary or anything else.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    background-size: cover;
+    background-repeat: no-repeat;
+    color: var(--secondary);
+    display: flex;
+    flex-direction: column;
+}
+
+i{
+    font-size: 2em;
+    color:var(--primary);
```
Comment (arjun-computer-geek) [inline style.css:29]: It's good to use color variables

**Review by arjun-computer-geek — APPROVED** [review]
Do some suggested change. Else look good.

**Pull request merged**